config/admins.txt

```diff
@@ -5,4 +5,5 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
-cassyjenelle - Host
\ No newline at end of file
+cassyjenelle - Host
+XanderDox - Host
```

Comment:
something's wrong, i can feel it

Reply:
just a feeling i've got, like somethings about to happen, but i don't know what

Reply:
Deleted the offending file. Thanks for catching this